mysql-ballerina/Config.toml

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+[sql]
+maxOpenConnections=10
+maxConnectionLifeTime=2000.5
+minIdleConnections=5
```

Comment:
Do we need this module ballerina package?

Reply:
Config.toml file is only needed in tests folder in mysql-ballerina module.So It can be removed from above folder location.

Reply:
Removed in https://github.com/ballerina-platform/module-ballerinax-mysql/pull/104/commits/d734425e49ba6508203c0ca11a8c8239a088b62d